jetty-servlet/src/test/java/org/eclipse/jetty/servlet/DispatcherTest.java

```diff
@@ -878,35 +1003,56 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         }
     }
 
-    public static class AssertForwardServlet extends HttpServlet implements Servlet
+    public static class DumpForwardServlet extends HttpServlet
```

Comment:
I made this generic, so it can be used from different flows.
It's a better idea to assert values in the same thread the test is running in anyway.